tests/integration.rs

```diff
@@ -2458,6 +2466,7 @@ mod tests {
         }
 
         #[test]
+        #[cfg(not(feature = "mshv"))]
         fn test_guest_numa_nodes() {
```

Comment:
Given the basic NUMA test is not working for MSHV, what about the `test_live_migration_numa`?